### ISSUE TITLE
Options List / Tab list doesn't show in Chrome latest release.

### ISSUE THREAD

**Issue by rizwank** [tabwrangler/tabwrangler#210]
Chrome 65.0.3325.146 (and slightly before) doesn't show the Tabwrangler history at all,. Right click on icon and going to options now goes to the Extension detail page, without any popup showing tabs. 

When I just click on the icon, nothing happens.

**Comment by ssorallen** [comment]
Thank you for the report. Could you help me debug what's going on so I can get it fixed?

1. Go to *Window* > *Extensions*
2. Toggle on the *Developer mode* switch in the upper right corner
3. Right click the Tab Wrangler icon in your URL bar
4. Choose *Inspect Popup*
5. Copy/paste any output you seen in the *Console* tab into this issue

**Comment by rizwank** [comment]
after step 3.

Inspect popup is not listed.
Inspect views- background is. clicking it and checking console does nothing.

when i right click the extension and select options, popup is listed under inspect views
nothing listed in console.

ty!

**Comment by rizwank** [comment]
Thanks. Works great on a new profile. 

And. (curses, but happy) works great on my profile now too!

It says 278 tabs wranglers, but the hint on the toolbar says 478...

**Comment by ssorallen** [comment]
> works great on my profile now too!

Excellent! 🎉

> It says 278 tabs wranglers, but the hint on the toolbar says 478...

When was the oldest tab in your Corral wrangled? The "tabs wrangled" feature was added some time ago, but prior to that it was not counting. If some of those 478 wrangled tabs were wrangled a long time ago, it's possible they are not in the count. It's also possible the counting is wrong... I haven't looked at that in a long time.

**Comment by rizwank** [comment]
It looks like it's exactly 200 below the top level amount. 

I restored an old one; the tabs wrangled section didn't reduce by one but the tooltip did.

**Comment by ssorallen** [comment]
> I restored an old one; the tabs wrangled section didn't reduce by one but the tooltip did.

Oh yes, that is working as I intended but I can see how the two are confusing. The badge on the icon is a count of the number of tabs in your list. The "tabs wrangled" count inside the popup only ever increases; it's a running count of how many tabs Tab Wrangler has closed since you installed the extension.

Would you expect it to work a different way?

I'm going to close this since everything appears to be working as expected now, but I'm happy to keep chatting and hearing how things might be improved. Let's open a new issue if there's something that sounds worth fixing or implementing.
